exporter/clickhousetracesexporter/writer.go

```diff
@@ -163,17 +163,10 @@ func (w *SpanWriter) writeBatch(batch []*Span) error {
 			return err
 		}
 	}
-	if w.attributeTable != "" {
+	if w.attributeTable != "" && w.attributeKeyTable != "" {
 		if err := w.writeTagBatch(batch); err != nil {
 			logBatch := batch[:int(math.Min(10, float64(len(batch))))]
-			w.logger.Error("Could not write a batch of spans to tag table: ", zap.Any("batch", logBatch), zap.Error(err))
-			return err
-		}
-	}
-	if w.attributeKeyTable != "" {
-		if err := w.writeTagKeyBatch(batch); err != nil {
-			logBatch := batch[:int(math.Min(10, float64(len(batch))))]
-			w.logger.Error("Could not write a batch of spans to tag key table: ", zap.Any("batch", logBatch), zap.Error(err))
+			w.logger.Error("Could not write a batch of spans to tag/tagKey tables: ", zap.Any("batch", logBatch), zap.Error(err))
```

Comment:
Put `zap.Error(err)` first so the user can see the error first before they have to scroll past the batch dump. How does logging a batch of spans help with the error here?

Reply:
> Put zap.Error(err) first so the user can see the error first before they have to scroll past the batch dump.

Done.

> How does logging a batch of spans help with the error here?

Debugging is easier if we know the spans. 
As we write to CH in batch and can't identify individual spans which caused error, so we need to log the whole batch if there's an error.